src/Yoti.Auth/AttributeConverter.cs

```diff
@@ -65,7 +24,7 @@ public static BaseAttribute ConvertToBaseAttribute(AttrpubapiV1.Attribute attrib
 
                 case ContentType.Date:
                     value = new YotiAttributeValue(TypeEnum.Date, attribute.Value.ToByteArray());
-                    return new YotiAttribute<DateTime>(
+                    return new YotiAttribute<DateTime?>(
```

Comment:
I don't think the DateTime needs to be nullable.
We can parse attribute values up front, as soon as we get the receipt.  If there is an error parsing any given DateTime, we log it, and we don't add that attribute to the profile.

Reply:
I've created https://lampkicking.atlassian.net/browse/SDK-679 to implement that

Reply:
Ok.  But I still don't see any need to change it to nullable in this PR, even if we do intend to take it out in a couple of days time.

Reply:
Ah sorry, didn't see what you meant. Changed now